projects/angular/components/ui-grid/src/managers/selection-manager.ts

```diff
@@ -34,6 +37,8 @@ export class SelectionManager<T extends IGridDataEntry> {
 
     public changed$: Subject<SelectionChange<T>> = new Subject();
 
+    public hasValue$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
Let's expose this as an Observable with `distictintUntilChanged()`
```suggestion
    private _hasValue$ = new BehaviorSubject<boolean>(false);
    public hasValue$ = this._hasValue$.pipe(distictintUntilChanged());
```